src/api/app/assets/stylesheets/webui/notifications.scss

```diff
@@ -0,0 +1,74 @@
+$notifications-filter-box-height: 3.5rem;
```

Comment:
I removed `.tasks-counter` and a bunch of `body.notifications-redesign`, then moved the file out of the `notifications_redesign` directory.